js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties

```diff
@@ -206,3 +206,11 @@ samlRoleListScopeConsentText=Meine Rollen
 offlineAccessScopeConsentText=Offline Zugriff
 profileScopeConsentText=Benutzerprofil
 emailScopeConsentText=E-Mail Adresse
+organizations=Organisationen
+organizationDescription=Unternehmen ansehen, denen du beigetreten bist
+emptyUserOrganizations=Keine Unternehmen
+emptyUserOrganizationsInstructions=Du bist bisher noch keinem Unternehmen beigetreten.
+searchOrganization=Nach Unternehmen suchen
+organizationList=Liste an Unternehmen
```

Comment:
IMHO we should translate it to "Organisation"/"Organisationen" consistently, and not switch to "Unternehmen" in some places.

Reply:
I would go with "Organisation".